ephios/plugins/files/forms.py

```diff
@@ -0,0 +1,50 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.forms import FileInput, Form, ModelForm, ModelMultipleChoiceField
+from django.template.defaultfilters import filesizeformat
+from django.utils.translation import gettext as _
+from django_select2.forms import Select2MultipleWidget
+
+from ephios.core.forms.events import BasePluginFormMixin
+from ephios.plugins.files.models import Document
+
+
+class DocumentForm(ModelForm):
+    class Meta:
+        model = Document
+        fields = ["title", "file"]
+        widgets = {"file": FileInput(attrs={"accept": ".pdf"})}
+
+    def clean_file(self):
+        file_size = self.cleaned_data["file"].size
+        if file_size > settings.GET_USERCONTENT_QUOTA():
+            raise ValidationError(
+                _("The file is too large. There are only %(quota)s available."),
+                params={"quota": filesizeformat(settings.GET_USERCONTENT_QUOTA())},
+            )
+
+
```

Comment:
I think we get a problem with duplicate filenames. Please check!

Reply:
django appends a random string to the filename if a file with the same name already exists